fhir-parser-resources/fhirabstractresource.py

```diff
@@ -110,7 +110,19 @@ def read_from(cls, path, server):
         instance = cls(jsondict=ret)
         instance.origin_server = server
         return instance
-    
+
+    def createUrl(self):
```

Comment:
Maybe a better name here is `createPath`? The return is not a url but a path used in urljoin later
https://github.com/smart-on-fhir/client-py/blob/b4da2bdc3bf024b6d0a78c85d80bf513302c1b13/fhirclient/server.py#L228